Cargo.toml

```diff
@@ -12,12 +12,12 @@ include = ["src/*.rs", "LICENSE-MIT", "LICENSE-APACHE", "Cargo.toml"]
 rust-version = "1.76"
 
 [dependencies]
-egui = { version = "0.28", default-features = false }
-serde_json = { version = "1.0.104", optional = true }
-simd-json = { version = "0.13.8", optional = true }
+egui = { version = "0.29", default-features = false }
+serde_json = { version = "1.0.128", optional = true }
+simd-json = { version = "0.14.0", optional = true }
```

Comment:
The `simd-json` and `serde_json` dependency versions should not need to be bumped here.

Reply:
This could force users to update their dependencies on those crates if their version was incompatible with the new version requirements.

Reply:
TBH these version requirements could be relaxed anyway, but happy to address that later and get the `egui` 0.29 update published.

Reply:
We could set serde-json to 1 

Other crates could also set to a more Open Version

Reply:
Updating dependencies iss Never Bad

Reply:
As a library, `egui_json_tree` should not force users to update their dependencies on those crates when `egui_json_tree` does not specifically require later versions.